factory/__main__.py

```diff
@@ -1,14 +1,16 @@
-import signal
 import argparse
-import sys
 import os
+import signal
+import sys
+from os import listdir
```

Comment:
can we use only the os package or its functions? This way we don't import the same thing twice

Reply:
Sure thing. Because some of the functions from the `os` library can be very long, such as:

```python
os.path.basedir(os.path.join(os.getcwd(), 'folder'))
```
> Exaggerated example

I think it would be best to import the functions from `os`.
Thoughts?